src/components/Select.vue

```diff
@@ -389,9 +389,10 @@
   import pointerScroll from '../mixins/pointerScroll'
   import typeAheadPointer from '../mixins/typeAheadPointer'
   import ajax from '../mixins/ajax'
+  import text from '../mixins/text'
```

Comment:
❗ Not tested, but should work in theory unless I have typoed something ❗
- Since [src/mixins/index.js ](https://github.com/sagalbot/vue-select/blob/master/src/mixins/index.js#L5)is already exporting them all, we can just import them at from once place to save space.
```suggestion
  import { ajax, pointerScroll,  text, pointer as typeAheadPointer  } from '../mixins'
```